pawan/html/html_assignment9.txt

```diff
@@ -0,0 +1,3 @@
+Q 9.1 If <span> and <div> are semantically neutral (that is, they do not confer any meaning to the content they envelop), then why would you even use them? What possible benefit could we get from using these elements in our web page?
+	answer:-The DIV and SPAN elements, in conjunction with the id and class attributes, offer a generic mechanism for adding structure to documents. These elements define content to be inline (SPAN) or block-level (DIV) but impose no other presentational idioms on the content. Thus, authors may use these elements in conjunction with style sheets, the lang attribute, etc., to tailor HTML to their own needs and tastes.
```

Comment:
Did you understand this, internet's answer?

Reply:
its depending on the user what he wants to use
 <span> can do only inline styling 
<div> can do a specific block styline

Reply:
Okay, that is correct but do you know more about "div" and "span", because the answer I expect is rely on that info.
Do you know about the widths of these two tags?
Every tag has some width to occupy on HTML page, some tags might don't have.
Read this topic first, then answer